modules/ROOT/pages/user-management.adoc

```diff
@@ -4,19 +4,52 @@
 :page-aliases: platform/user-management.adoc
 
 User management is a feature within Aura that allows admins to invite users and set their roles within an isolated environment.
+Each project can have multiple users with individual accounts allowing access to the same environment.
 
-[NOTE]
-====
-To invite users, you must perform the action at the project level.
-Go to *Project > Users* to send an invitation.
-When you invite someone at the project level, they are automatically added to the organization as an `ORG_MEMBER`.
-After the user accepts the invite, you can optionally update their organization-level role via *Organization > Users*.
-However, editing an organization role is not required unless you want to grant them specific organization-wide permissions.
-====
+== Invite users
+
+As an _Admin_, to invite a new user, you invite at the project level and assign a project-level role.
+
+. Within a project, go to *Users* and select *Invite user*.
+. Enter the **Email** address of the person you want to invite and press *Add*.
+. Select the user's **Role**.
+. Select **Invite**.
+* The invited user will receive an email with a link to accept the invitation.
+* The new user will appear within the list of users on the project **User** page with the `Pending invite` Status until they accept the invite.
+* On accepting the invite, the invited user automatically gets an `ORG_MEMBER` role in the organization the project is part of
+* If needed, you can change the org-level role after the invite is accepted - only do this if the user needs a different role at the organization level.
+
+.Grant users access to a project
+image::inviteusers.png[]
+
+== Accept an invite
+
+When invited to a project, you will receive an email with a link to accept the invite.
+This link will direct you to the Aura console, where a **Project invitation** modal will appear.
+You can select the project(s) you have been invited to and choose to accept or decline the invite(s).
+
+// You can also close the **Project invitation** modal without accepting or declining the invite(s) and later manually re-open the modal by selecting the **Pending invites** envelope icon in the console header.
+
+== Edit roles
+
+As an _Admin_, to edit an existing user's role:
+
+. Select the more actions (three dots) icon next to the user's name from the **User** page.
+. Select the user's new **Role**.
+. Select **Save**.
+
+== Delete users
+
+As an _Admin_, to delete an existing user:
+
+. Select the more actions (three dots) next to the user's name from the **User** page.
+. Select **Delete**.
```

Comment:
```suggestion
```
Can be included in the section above. We must really try not to teach users how to click buttons.